app/src/main/res/layout/fragment_amortisationsrechnerpv.xml

```diff
@@ -128,8 +125,8 @@
                     android:text="@string/amortPV_text_verguetung"
                     android:textSize="@dimen/content_text_normal"
                     app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintStart_toStartOf="@+id/amortPV_text_vergütung_beschreibung"
-                    app:layout_constraintTop_toBottomOf="@+id/amortPV_edit_kWh_pro_kWp" />
+                    app:layout_constraintStart_toStartOf="@+id/amortPV_text_ertrag_beschreibung"
+                    app:layout_constraintTop_toBottomOf="@+id/amortPV_edit_ertrag" />
```

Comment:
XML naming and ids should be in snake case only.

Reply:
This will be fixed in the Pull Request from AmortisationsrechnerPV.